thingsboard_gateway/gateway/tb_gateway_service.py

```diff
@@ -529,7 +530,7 @@ def __rpc_gateway_processing(self, request_id, content):
         arguments = content.get('params', {})
         if content.get("timeout") is not None:
             arguments.update({"timeout": content["timeout"]})
-        method_to_call = content["method"].replace("gateway_", "")
+        method_to_call = content["method"]
```

Comment:
This part of the code relates to RPC request to the gateway device and we pass the full message to this function, so we should remove "gateway_" prefix. Please take a look what contains "content" variable.

Reply:
So are you saying that the problem with this change is that the "gateway_" prefix is no longer removed?  What I wanted to achieve with this change was to remove the module from the content's method earlier so that it would not be passed to the server_side_rpc_handler of a gateway.  Unless I am mistaken, the "gateway_" prefix is still removed.  

For example, if the method of the content started as "gateway_ping", then the method would be split on line 498 into "gateway" and "ping."  Then on line 502, I set the method of the content to "ping."  Therefore, when content is passed to __rpc_gateway_processing, the prefix is already removed.

If this change is still not wanted let me know and I will remove it so that only the connector type issue is fixed.  Sorry for the delayed response, school has been very busy.